lib/wind fails to build on modern OSes

**Describe the bug**
The wind library fails to build on modern OSes (such as ubuntu20, centos8) because "python" no longer exists.

**To Reproduce**
Build heimdal

**Expected behavior**
Success

**Actual behavior**
Failure

```
make[4]: Entering directory '/home/build/git/sold/sold-master/ubuntu20-amd64/build/heimdal.amd64/lib/wind'
test -f bidi_table.c || python /home/build/git/sold/sold-master/heimdal/lib/wind/gen-bidi.py /home/build/git/sold/sold-master/heimdal/lib/wind/rfc3454.txt .
/bin/bash: python: command not found
make[4]: *** [Makefile:1769: bidi_table.c] Error 127
make[4]: Leaving directory '/home/build/git/sold/sold-master/ubuntu20-amd64/build/heimdal.amd64/lib/wind'
make[3]: *** [Makefile:835: all] Error 2
make[3]: Leaving directory '/home/build/git/sold/sold-master/ubuntu20-amd64/build/heimdal.amd64/lib/wind'
make[2]: *** [Makefile:565: all-recursive] Error 1
make[2]: Leaving directory '/home/build/git/sold/sold-master/ubuntu20-amd64/build/heimdal.amd64/lib'
make[1]: *** [Makefile:613: all-recursive] Error 1
make[1]: Leaving directory '/home/build/git/sold/sold-master/ubuntu20-amd64/build/heimdal.amd64'
make: *** [/home/build/git/sold/sold-master/heimdal.make:228: inter-build-heimdal.amd64] Error 2
```

--- a/configure.ac
+++ b/configure.ac
@@ -18,6 +18,7 @@ dnl Checks for programs.
 AC_PROG_CC
 AM_PROG_CC_C_O
 AC_PROG_CPP
+AM_PATH_PYTHON
 AC_CHECK_PROG(CLANG_FORMAT, clang-format, [clang-format], [no])
 
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])

--- a/lib/wind/Makefile.am
+++ b/lib/wind/Makefile.am
@@ -89,8 +89,6 @@ idn_lookup_SOURCES = idn-lookup.c
 
 LDADD = libwind.la $(LIB_roken)
 
-PYTHON = python
-
 if !MAINTAINER_MODE
 skip_python = test -f $@ ||
 endif

